src/sagemaker/serve/builder/model_builder.py

```diff
@@ -672,3 +682,58 @@ def validate(self, model_dir: str) -> Type[bool]:
         """
 
         return get_metadata(model_dir)
+
+    def total_inference_model_size_mib(self):
```

Comment:
internal functions should start with underscore

Reply:
Ack wil do, thanks